test/Tester/HostedTestClusterBase.cs

```diff
@@ -14,7 +14,7 @@ public abstract class HostedTestClusterEnsureDefaultStarted : OrleansTestingBase
     {
         protected TestCluster HostedCluster { get; private set; }
 
-        public HostedTestClusterEnsureDefaultStarted(DefaultClusterFixture fixture)
+        public HostedTestClusterEnsureDefaultStarted(BaseTestClusterFixture fixture)
```

Comment:
Also, I'm not sure why? It only accesses HostCluster which is defined on BaseTestClassFixture